### PR TITLE
api correction

### PR DESCRIPTION
**Pull request by jaworskak**

### CODE REVIEW

File: backend/no_waste.py [hunk 1]
```diff
@@ -26,24 +26,48 @@ class Recipe(db.Model):
     recipe_name = db.Column(db.String(255))
     recipe = db.Column(db.Text)
 
+    @property
+    def serialize(self):
+        return {
+            "title": self.recipe_name,
+            "imgUrl": None,
```
Comment (jakule) [inline backend/no_waste.py:33]: You can also use something like this https://loremflickr.com/

File: backend/no_waste.py [hunk 2]
```diff
@@ -26,24 +26,48 @@ class Recipe(db.Model):
...
+            "description": self.recipe,
+            "ingredients": ["Czekolada", "Chleb"]
```
Comment (jakule) [inline backend/no_waste.py:35]: I know this is just a stub, but c`mon. All other properties are set where they should be.

File: backend/no_waste.py [hunk 3]
```diff
@@ -26,24 +26,48 @@ class Recipe(db.Model):
...
+
+
 
 db.create_all()
 
 if Recipe.query.count() == 0:
     # insert initial state
+    Recipe.query.delete()
+    Product.query.delete()
     recipe = Recipe(id=1, recipe_name="Kielbasa", recipe="Przepis na kielbase")
```
Comment (jakule) [inline backend/no_waste.py:52]: What do you think about this?
```python
recipes = [
    Recipe(id=1, recipe_name="Kielbasa", recipe="Przepis na kielbase"),
    Recipe(id=2, recipe_name="Chleb", recipe="Przepis na chleb"),
    Recipe(id=3, recipe_name="Piwo", recipe="Przepis na piwo"),
    Product(id=1, product_name="Ziemniak"),
    Product(id=2, product_name="Kielbasa"),
    Product(id=3, product_name="Pomidor"),
]

db.session.add_all(recipes)
db.commit()
```

**Review by jakule — CHANGES_REQUESTED** [review]

File: backend/no_waste.py [hunk 4]
```diff
@@ -52,9 +76,13 @@ def recipe(recipe_id: int):
     recipe = Recipe.query.get_or_404(recipe_id)  # get elem by id
 
     return jsonify(
-        {"id": recipe.id, "recipe_name": recipe.recipe_name, "recipe": recipe.recipe,}
+        {"id": recipe.id, "recipe_name": recipe.recipe_name, "recipe": recipe.recipe, }
     )
 
+# get list of products
+@app.route("/product")
```
Comment (jakule) [inline backend/no_waste.py:83]: It should be `products`

File: backend/no_waste.py [hunk 5]
```diff
@@ -52,9 +76,13 @@ def recipe(recipe_id: int):
     recipe = Recipe.query.get_or_404(recipe_id)  # get elem by id
 
     return jsonify(
-        {"id": recipe.id, "recipe_name": recipe.recipe_name, "recipe": recipe.recipe,}
+        {"id": recipe.id, "recipe_name": recipe.recipe_name, "recipe": recipe.recipe, }
```
Comment (jakule) [inline backend/no_waste.py:79]: Why the format here is different from `/recipes` endpoint? I think we should use the serialize method after you implemented it already.

**Review by jakule — APPROVED** [review]
LGTM

File: backend/no_waste.py [hunk 6]
```diff
@@ -25,36 +25,58 @@ class Recipe(db.Model):
...
+
+    db.session.add_all(recipes)
     db.session.commit()
 
 
 # all recipes in database
 @app.route("/recipes")
 def recipelist():
-    return jsonify([recipes.recipe_name for recipes in Recipe.query.all()])
+    return jsonify([x.serialize for x in Recipe.query.all()])
```
Comment (AnnaZalewska) [inline backend/no_waste.py:66]: why x?

Comment (jaworskak) [reply backend/no_waste.py:66]: Kuba recommended using serialize in both requests.
Beside that i thought that I should return all not only name

Comment (AnnaZalewska) [reply backend/no_waste.py:66]: I ask why you use name x. Recipes was good name.

**Review by AnnaZalewska — APPROVED** [review]

**Pull request merged**